dist/Data-Dumper/Dumper.pm

```diff
@@ -394,7 +394,9 @@ sub _dump {
         else {
           $pat = "$val";
         }
-        $pat =~ s <(\\.)|/> { $1 || '\\/' }ge;
+        $pat =~ s <(\\.)|(/)|([^[:ascii:]])> {
+            $1 // (defined $2 ? '\\/' : sprintf '\x{%x}', ord $3)
```

Comment:
This seems like an unnecessarily complicated way to do this? The above line does an "escape `/` unless it's escaped", we don't need to do any such tricks.

AFAICT we can just do `$pat =~ s/([^[:ascii:]])/sprintf '\x{%x}', ord $1/ge`